impl/pom.xml

```diff
@@ -953,8 +953,8 @@
             </activation>
             <build>
                 <plugins>
-
                     <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
                             <excludePackageNames>org.apache.myfaces.ee6</excludePackageNames>
```

Comment:
Thomas removed this, but I'm not sure if I should? 

https://github.com/apache/myfaces/commit/1dbfe09efabfe174f8ec65f48d98fdec1e996fa4#diff-abb4e0d98dcc6e2742e9b46d0f75f831b0e0dbe1feac6a85ebf38167130528a9L873-L875